### PR TITLE
add service check to why-did-my-monitor-settings-change-not-take-effe…

### PR DESCRIPTION
**Pull request by djchinia**
### What does this PR do?

add service check alongside host monitor in first sentence 

### Motivation

https://datadoghq.atlassian.net/browse/MNTS-89720?focusedCommentId=1064397

### Additional Notes

https://docs-staging.datadoghq.com/dan.chiniara/add-service-check-why-did-my-monitor-settings-change-not-take-effect/monitors/guide/why-did-my-monitor-settings-change-not-take-effect/

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/monitors/guide/why-did-my-monitor-settings-change-not-take-effect.md [hunk 1]
```diff
@@ -5,6 +5,6 @@ aliases:
 - /monitors/faq/why-did-my-monitor-settings-change-not-take-effect
 ---
 
-Datadog keeps monitor groups available in the UI for 24 hours (48h for host monitors) unless the query is changed. If you do not have *No Data* alert settings enabled and your group for a metric monitor stops reporting data, it will persist on the monitor status page until it ages out, though that group will stop being evaluated after a short absence (specific timing depends on your settings).
+Datadog keeps monitor groups available in the UI for 24 hours (48h for service check that notify on *No Data* and host monitors) unless the query is changed. If you do not have *No Data* alert settings enabled and your group for a metric monitor stops reporting data, it will persist on the monitor status page until it ages out, though that group will stop being evaluated after a short absence (specific timing depends on your settings).
```
Comment (hestonhoffman) [inline content/en/monitors/guide/why-did-my-monitor-settings-change-not-take-effect.md:8]:
```suggestion
Datadog keeps monitor groups available in the UI for 24 hours unless the query is changed. Host monitors and service checks that notify on on *No Data* are available for 48 hours. If you do not have *No Data* alert settings enabled and your group for a metric monitor stops reporting data, the group persists on the monitor status page until it ages out, though that group stops being evaluated after a short absence. The specific timing for how long the group persists depends on your settings.
```

File: content/en/monitors/guide/why-did-my-monitor-settings-change-not-take-effect.md [hunk 2]
```diff
@@ -5,6 +5,6 @@ aliases:
...
 
 For event monitors, however, Datadog also keeps groups for evaluations for at least 24 hours. This means that if a monitor is updated and the groups are changed in the query, some old groups may persist. If you must change the group settings on your event monitor, you may want clone or create a new monitor to reflect your new groups.  Alternatively, you can mute them if you would like to maintain the monitor but silence any alerts that would result from the changes.
```
Comment (hestonhoffman) [inline content/en/monitors/guide/why-did-my-monitor-settings-change-not-take-effect.md:10]:
```suggestion
For event monitors, however, Datadog also keeps groups for evaluations for at least 24 hours. This means that if a monitor is updated and the groups are changed in the query, some old groups may persist. If you must change the group settings on your event monitor, you may want clone or create a new monitor to reflect your new groups. Alternatively, you can mute them if you would like to maintain the monitor but silence any alerts that would result from the changes.
```

Comment (hestonhoffman) [inline content/en/monitors/guide/why-did-my-monitor-settings-change-not-take-effect.md:10]:
```suggestion
For event monitors, however, Datadog also keeps groups for evaluations for at least 24 hours. This means that if a monitor is updated and the groups are changed in the query, some old groups may persist. If you must change the group settings on your event monitor, you may want clone or create a new monitor to reflect your new groups. Alternatively, you can mute them if you would like to maintain the monitor but silence any alerts that would result from the changes.
```

Comment (djchinia) [reply content/en/monitors/guide/why-did-my-monitor-settings-change-not-take-effect.md:10]: Thanks! That reads much better.

File: content/en/monitors/guide/why-did-my-monitor-settings-change-not-take-effect.md [hunk 3]
```diff
@@ -5,6 +5,6 @@ aliases:
 - /monitors/faq/why-did-my-monitor-settings-change-not-take-effect
 ---
 
-Datadog keeps monitor groups available in the UI for 24 hours (48h for host monitors) unless the query is changed. If you do not have *No Data* alert settings enabled and your group for a metric monitor stops reporting data, it will persist on the monitor status page until it ages out, though that group will stop being evaluated after a short absence (specific timing depends on your settings).
+Datadog keeps monitor groups available in the UI for 24 hours unless the query is changed. Host monitors and service checks that notify on on *No Data* are available for 48 hours. If you do not have *No Data* alert settings enabled and your group for a metric monitor stops reporting data, the group persists on the monitor status page until it ages out, though that group stops being evaluated after a short absence. The specific timing for how long the group persists depends on your settings.
```
Comment (hestonhoffman) [inline content/en/monitors/guide/why-did-my-monitor-settings-change-not-take-effect.md:8]:
```suggestion
Datadog keeps monitor groups available in the UI for 24 hours unless the query is changed. Host monitors and service checks that notify on *No Data* are available for 48 hours. If you do not have *No Data* alert settings enabled and your group for a metric monitor stops reporting data, the group persists on the monitor status page until it ages out, though that group stops being evaluated after a short absence. The specific timing for how long the group persists depends on your settings.
```

**Review by hestonhoffman — APPROVED** [review]

**Pull request merged**